requests_toolbelt/adapters/appengine.py

```diff
@@ -27,6 +31,7 @@ class AppEngineAdapter(adapters.HTTPAdapter):
         >>> from requests_toolbelt.adapters import appengine
         >>> s = requests.Session()
         >>> if using_appengine():
```

Comment:
Why not use the real function [`is_appengine_sandbox`](https://github.com/shazow/urllib3/blob/master/urllib3/contrib/appengine.py#L215)? IMO, it's better to have an example that works.

(Don't let this block though, I can always submit a PR to update docstrings)